generate_mosfet_dual.py

```diff
@@ -184,75 +185,273 @@ def generate_dev(
-            FetConfig('DMC3021LSD[Q]', 30, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], [
-                'https://www.diodes.com/assets/Datasheets/ds32152.pdf',
-                'https://www.diodes.com/assets/Datasheets/DMC3021LSDQ.pdf',
-            ]),
-            FetConfig('DMC3025LSD[Q]', 30, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], [
-                'https://www.diodes.com/assets/Datasheets/DMC3025LSD.pdf',
-                'https://www.diodes.com/assets/Datasheets/DMC3025LSDQ.pdf',
-            ]),
-            FetConfig('DMC3028LSD[Q[X]]', 30, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], [
-                'https://www.diodes.com/assets/Datasheets/DMC3028LSD.pdf',
-                'https://www.diodes.com/assets/Datasheets/DMC3028LSDX.pdf',
-                'https://www.diodes.com/assets/Datasheets/DMC3028LSDXQ.pdf',
-            ]),
-            FetConfig('DMC3032LSD', 30, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], 'https://www.diodes.com/assets/Datasheets/ds32153.pdf'),
-            FetConfig('DMC4015SSD', 40, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], 'https://www.diodes.com/assets/Datasheets/DMC4015SSD.pdf'),
-            FetConfig('DMC4028SSD', 40, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], 'https://www.diodes.com/assets/Datasheets/DMC4028SSD.pdf'),
-            FetConfig('DMC4029SSD', 40, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], 'https://www.diodes.com/assets/Datasheets/DMC4029SSD.pdf'),
-            FetConfig('DMC4040SSD[Q]', 40, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], [
-                'https://www.diodes.com/assets/Datasheets/ds32120.pdf',
-                'https://www.diodes.com/assets/Datasheets/DMC4040SSDQ.pdf',
-            ]),
-            FetConfig('DMC4047LSD', 40, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], 'https://www.diodes.com/assets/Datasheets/DMC4047LSD.pdf'),
-            FetConfig('DMC4050SSD[Q]', 40, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], 'https://www.diodes.com/assets/Datasheets/DS33310.pdf'),
-            FetConfig('DMC6040SSD[Q]', 60, 'SOIC127P600X175-8', [
-                'sn', 'gn', 'sp', 'gp', 'dp', 'dp', 'dn', 'dn',
-            ], [
-                'https://www.diodes.com/assets/Datasheets/DMC6040SSD.pdf',
-                'https://www.diodes.com/assets/Datasheets/DMC6040SSDQ.pdf',
-            ]),
-
+            FetConfig(
```

Comment:
In places like these I'd suppress the formatter... Is there a way to disable ruff for a block of code (using a stop / start command)?

Reply:
One can just add `# fmt: off` / `on` pairs.